src/test/java/ca/uhn/fhir/jpa/starter/OperationCollectDataTest.java

```diff
@@ -168,7 +295,27 @@ private void postResource(String theUrl, String filePath)
 		}
 	}
 
-	private String readJsonFile(String path) throws IOException {
+	private CloseableHttpResponse postXmlResource(String theUrl, String filePath)
+      throws IOException{
+    HttpPost post = new HttpPost(theUrl);
+    String xml = readFile(filePath);
+
```

Comment:
Also thei , Create one Post in a given privae method and re-use that with the existing tests